CompetitionMARS/Tests/CertificationTests.cs

```diff
@@ -0,0 +1,364 @@
+            certPage.AddNewCertification(cert.CertificateOrAward, cert.CertifiedFrom, cert.CertificationYear);
+            // Track the added CertificateName for cleanup
+            //addedCertificationsEntries.Add(cert.CertificateOrAward);
+            // Verify that the system adds the entry without crashing
+            var validationMessage = driver.FindElement(By.XPath($"//div[contains(text(),'{cert.CertificateOrAward}')]"));
+            Assert.IsTrue(validationMessage.Displayed, "System crashed due to large payload in Degree field");
+            if (true)
+            {
+                test.Log(Status.Pass, "success");
+            }
+            else
+            {
+                test.Log(Status.Fail, "failed");
+            }
+        }
+        [Test, Order(11)]
+        public void AddCertificationn_MultipleEntries()
+        {
+            ExtentTest test = extent.CreateTest("AddCertification_MultipleEntries");
+
+            CertificationData cert = certPage.GetCertificationData("validdata4");
+
+            // Add certification entry 1
+            certPage.AddNewCertification(cert.CertificateOrAward, cert.CertifiedFrom, cert.CertificationYear);
+            
+            bool isCertificationAdded = certPage.IsCertificationAdded(cert.CertificateOrAward);
+            Assert.IsTrue(isCertificationAdded, "Certification entry was not added successfully.");
+
+            //Add certification entry 2
+            CertificationData cert2 = certPage.GetCertificationData("validdata5");
+            certPage.AddNewCertification(cert2.CertificateOrAward, cert2.CertifiedFrom, cert2.CertificationYear);
+            
+            bool isCertificationAdded2 = certPage.IsCertificationAdded(cert.CertificateOrAward);
+            Assert.IsTrue(isCertificationAdded, "Certification entry was not added successfully.");
+
+            //Add certification entry 3
+            CertificationData cert3 = certPage.GetCertificationData("validdata6");
+            certPage.AddNewCertification(cert3.CertificateOrAward, cert3.CertifiedFrom, cert3.CertificationYear);
+            
+            bool isCertificationAdded3 = certPage.IsCertificationAdded(cert.CertificateOrAward);
+            Assert.IsTrue(isCertificationAdded, "Certification entry was not added successfully.");
+            // Log the result
+            if (true)
+            {
+                test.Log(Status.Pass, "All multiple certification entries were added successfully.");
+            }
+            else
+            {
+                test.Log(Status.Fail, "failed");
+            }
+
+        }
+
+      
+
+
+            /*[TearDown]
+            public void CleanData()
+            {
+                CleanupAddedCertificationEntries();
+            }*/
+    [OneTimeTearDown]
+    public void TearDown()
+        {
+            extent.Flush(); // End the report
+            certPage.CleanupAddedCertificationEntries();
```

Comment:
ideally you should do a clean up of all data in one time setup and then after each test clean the data that your test has add only and not everything